test/verify/check-system-services

```diff
@@ -382,6 +391,39 @@ Unit=test.service
         b.click(self.svc_sel("systemd-halt.service"))
         b.wait_text('.cockpit-log-panel .pf-c-card__body', "No log entries")
 
+        b.go("#/")
+
+        self.write_file("/tmp/mem-hog.awk", 'BEGIN { system("while [ ! -f /tmp/continue ]; do sleep 1; done"); y = sprintf("%50000000s",""); system("sleep infinity") }')
+        self.write_file("/etc/systemd/system/mem-hog.service",
+                        """
+[Unit]
+Description=Mem Hog Service
+
+[Service]
+ExecStart=/bin/awk -f /tmp/mem-hog.awk
+
+[Install]
+WantedBy=default.target
+""")
+
+        # After writing files out tell systemd about them
+        m.execute("systemctl daemon-reload")
+
+        # Check that MemoryCurrent is shown
+        b.click(self.svc_sel("mem-hog.service"))
+        b.wait_not_present("#memory")
+        m.execute("systemctl start mem-hog.service")
+        initial_memory = float(b.text("#memory .pf-c-description-list__text").split(" ")[0])
+        self.assertGreater(initial_memory, 1)
+        m.execute("touch /tmp/continue")
+        # MemoryCurrent auto-updates every 30s - when it updates the memory used should be ~50MiB
+        with b.wait_timeout(30):
```

Comment:
That would fail if the JS code's interval timer happened to start at the same time as the test -- which is actually fairly likely. Let's just use the standard timeout here.